spring-cloud-gcp-samples/spring-cloud-gcp-data-jpa-sample/pom.xml

```diff
@@ -20,7 +20,15 @@
 
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-gcp-starter-sql</artifactId>
+			<artifactId>spring-cloud-gcp-autoconfigure</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.google.cloud.sql</groupId>
```

Comment:
As I mentioned in https://github.com/spring-cloud/spring-cloud-gcp/issues/321, I do think there's value in having a starter for bringing in the right dependencies (along with their correct versions). Wouldn't you call that an opinionated starter?